hack/update-codegen.sh

```diff
@@ -52,7 +52,7 @@ function git_find() {
     # Similar to find but faster and easier to understand.  We want to include
     # modified and untracked files because this might be running against code
     # which is not tracked by git yet.
-    git ls-files -cmo --exclude-standard "$@"
+    git ls-files -cmo --exclude-standard ':!:vendor/*' "$@"
 }
 
 function git_grep() {
```

Comment:
same issue in git_grep?

Reply:
I think it's less of a problem since we only grep through `--untracked` files.

Reply:
No, `--untracked` means ALSO grep untracked files.

That said, the one place that calls `git_grep` without an explict root path already filters out vendor, so it's not *as* broken.  It seems reasonable to me to move that into `git_grep` and `git_find` - I can't think why this script ever wants to touch vendor/

Reply:
I've also looked into how the invocation for `git grep`, and pathspecs are only after `--` so we'd need to have the ability to pass arguments before and after the double dash. And just like you're saying that single invocation already handles it as necessary.

Reply:
Updated.